tests/sentry/models/test_release.py

```diff
@@ -847,6 +847,41 @@ def test_parse_release_into_semver_cols_with_get_or_create(self):
         assert release.build_number is None
         assert release.package == "org.example.FooApp"
 
+    def test_parse_release_into_semver_cols_on_pre_save(self):
+        """
+        Test that ensures get_or_create populates semver fields
```

Comment:
Nit: These doc strings look wrong

Reply:
oops you are right.. I had it on my mind to change but forgot